app/controllers/api/v1/project_roles_controller.rb

```diff
@@ -8,4 +8,20 @@ class Api::V1::ProjectRolesController < Api::ApiController
   def resource_name
     "project_role"
   end
+
+  def update
+    super do
+      Mailers::UserAddedToProject.run!(api_user: api_user, resource_id: params[:id], roles: roles) if check_new_roles(roles).present?
+    end
+  end
+
+  private
+
+  def roles
+    params[:project_roles][:roles]
+  end
+
+  def check_new_roles(roles)
```

Comment:
make this method return what you want..i..e push the present into the method and return the boolean, so change the name to `new_roles_present(roles)?` or something...